actions/cli/readme.md

```diff
@@ -0,0 +1,23 @@
+# Ignite CLI Action
+
+This action makes the `ignite` CLI available as a Github Action.
```

Comment:
can you update comment to be related to elys? (I think using `ignite` word would not be ideal here)

Reply:
this is just a fork of the original folder from the ignite-cli repo